diamm/helpers/object_pagination.py

```diff
@@ -1,9 +1,9 @@
-from collections import OrderedDict
 import re
-
+from collections import OrderedDict
```

Comment:
I would agree but I would also argue that grouping them by std-lib, third-party, and local imports makes it easier to quickly understand what's being included. That being said, I see the existing convention everywhere else so I'll stick to that